cli.js

```diff
@@ -711,6 +712,10 @@ function codegen(json, result) {
   let avoJsonTask = writeAvoJson(newJson);
 
   Promise.all(_.concat([avoJsonTask], sourceTasks)).then(() => {
+    if (errors !== undefined && errors !== null && errors !== "") {
+      report.warn(errors + "\n");
```

Comment:
Was it not showing errors at all before?

Reply:
We didn't allow you to get this far if you had any errors, we aborted the codegen and only logged the errors. Here we're running the codegen, but also showing you the errors (the `--force` case)